### ISSUE TITLE
Log tailing fails with a Guzzle fatal error

### ISSUE THREAD

**Issue by anavarre** [acquia/cli#822]
```
test:~/project (main) $ acli log:tail

 Please select a Cloud Platform environment: [dev]:
  [0] dev
  [1] ode28
  [2] ode29
  [3] prod
  [4] test
 > 0

 Please select one or more logs as a comma-separated list::
  [0] Apache access
  [1] Apache error
  [2] Drupal request
  [3] Drupal watchdog
  [4] Fpm access
  [5] Fpm error
  [6] PHP error
  [7] Newrelic php agent
  [8] MySQL slow query
 > 0

Streaming has started and new logs will appear below. Use Ctrl+C to exit.

Fatal error: Uncaught Error: Call to undefined function GuzzleHttp\Psr7\uri_for() in phar:///opt/local/acli/vendor/ratchet/pawl/src/Connector.php:126
Stack trace:
#0 phar:///opt/local/acli/vendor/ratchet/pawl/src/Connector.php(38): Ratchet\Client\Connector->generateRequest()
#1 phar:///opt/local/acli/vendor/typhonius/acquia-logstream/src/LogstreamManager.php(195): Ratchet\Client\Connector->__invoke()
#2 phar:///opt/local/acli/src/Command/LogTailCommand.php(45): AcquiaLogstream\LogstreamManager->stream()
#3 phar:///opt/local/acli/vendor/symfony/console/Command/Command.php(298): Acquia\Cli\Command\LogTailCommand->execute()
#4 phar:///opt/local/acli/src/Command/CommandBase.php(402): Symfony\Component\Console\Command\Command->run()
#5 phar:///opt/local/acli/vendor/symfony/console/Application.php(1023): Acquia\Cli\Command\CommandBase->run()
#6 phar:///opt/local/acli/vendor/symfony/console/Application.php(299): Symfony\Component\Console\Application->doRunCommand()
#7 phar:///opt/local/acli/vendor/symfony/console/Application.php in phar:///opt/local/acli/vendor/ratchet/pawl/src/Connector.php on line 126
```

Happening with the latest version

```
test:~/project (main) $ acli --version
Acquia CLI 1.22.0
```

**Comment by grasmash** [comment]
Need this merged: https://github.com/typhonius/acquia-logstream/pull/132

**Comment by anavarre** [comment]
https://github.com/typhonius/acquia-logstream/pull/132 is merged, we're unblocked.

**Comment by danepowell** [comment]
Can y'all see if this fixes it? https://github.com/acquia/cli/pull/869

**Comment by anavarre** [comment]
I can see there's no longer any error, so the fix works 👌 . However I can't explain why logs aren't streaming, which is a different issue that is out of scope here.
